prov/rxm/src/rxm_domain.c

```diff
@@ -837,6 +837,23 @@ static void rxm_config_dyn_rbuf(struct rxm_domain *domain, struct fi_info *info,
 	}
 }
 
+static uint64_t get_capabilities_mask(struct fid_domain *domain)
+{
+	struct fi_collective_attr attr;
+	uint64_t mask = 0;
+	
+	attr.op = FI_MIN;
+	attr.datatype = FI_INT8;
+	attr.datatype_attr.count =1;
+	attr.datatype_attr.size =1;
```

Comment:
This would be slightly more descriptive as ```= sizeof(int8_t)```

Reply:
Done.